contracts/RMRK/extension/tokenProperties/RMRKTokenPropertiesRepository.sol

```diff
@@ -451,6 +689,188 @@ contract RMRKTokenPropertiesRepository is IRMRKTokenPropertiesRepository {
         }
     }
 
+    /**
+     * @notice Used to get multiple sting parameter values for a token.
+     * @dev The `StringProperty` struct contains the following fields:
+     *  [
+     *     string key,
+     *     string value
+     *  ]
+     * @param collection Address of the collection the token belongs to
+     * @param tokenId ID of the token for which the properties are being retrieved
+     * @param stringKeys An array of string keys to retrieve
+     * @return An array of `StringProperty` structs
+     */
+    function _getStringTokenProperties(
```

Comment:
Why not expose these getters? They're already there and are probably cheaper if you want only properties of the same type. Thinking of cases where other contracts use a view but affect the state, gas would be important there. :snake: :wink:

Reply:
Do you think that exposed multi-getters are beneficial? I'm not against it, but I added them only because the current master multi-getter can't operate without them 😅

Reply:
I totally do and gave you my reasons. If only need ints I'm pretty sure a specific getter will be cheaper.